doc/doxygen/references.bib

```diff
@@ -1294,13 +1294,18 @@ @article{mitchell2014hp
 % Step 76
 % ------------------------------------
 
-@misc{munch2020hyperdeal,
-  author =        {P. Munch and K. Kormann and M. Kronbichler},
-  title =         {hyper.deal: An efficient, matrix-free finite-element library for high-dimensional partial differential equations},
-  year =          {2020},
-  eprint =        {2002.08110},
-  archivePrefix = {arXiv},
-  primaryClass =  {cs.MS}
+@article{munch2020hyperdeal,
+  doi = {10.1145/3469720},
+  url = {https://dl.acm.org/doi/abs/10.1145/3469720},
+  year = 2021,
+  month = sep,
+  publisher = {Association for Computing Machinery ({ACM})},
+  volume = 47,
+  number = 4,
+  pages = {1--34},
```

Comment:
You will need some sort of article number.

Reply:
Interesting, `doi2bib` did not provide this information.

Clicking on the cite button on the journal webpage gives `articleno` as a field. This is not bibtex canon. For example, the [`plain.bst` style](http://mirrors.ctan.org/biblio/bibtex/base/plain.bst) does not provide that field.

Shall we replace the `pages` field instead, i.e., `pages = 33` in this case?

Reply:
This entry in our bibliogrpahy for example uses `articleno`.
https://github.com/dealii/dealii/blob/8bfb9e576afe8ab7228d2686c0acd8b9dafb133b/doc/doxygen/references.bib#L1951-L1960

But that information is not displayed on our website:
https://www.dealii.org/developer/doxygen/deal.II/citelist.html#CITEREF_ljungkvist2017matrix

Reply:
We could maybe comprimise on `pages = {33/1--34}`. I like this.